packages/autorest/eng/ci.yml

```diff
@@ -0,0 +1,190 @@
+trigger:
+  - v6
+
+parameters:
```

Comment:
I think we can remove this eng folder for now and bring it back later on. If we leave this in .scripts in the root we'll have less moving parts for this initial refator

Reply:
removed.